recipes/goodvibes/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "goodvibes" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea2ab9cea80a1f57ead3fb6135450ecc6324923f987038210453458093078753
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}  # not sure if you need this one
+  host:
+    - pip
+    - python
+  run:
+    - cclib
+    - numpy
+    - python
+
+test:
+  imports:
+    - goodvibes
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/patonlab/goodvibes
+  license: MIT
+  license_file: 
+    - LICENSE.txt # GoodVibes license
+    - LICENSE.txt # Symmetry license
```

Comment:
Might be better to rename Symmetry's license file. I am not sure how conda build behaves when two filenames collide like this, even if across directories.

Reply:
I see. I will just update and do a GV release now, that way I don't forget about it down the road and then the licenses are all correct for conda-forge